yt_dlp/YoutubeDL.py

```diff
@@ -3327,6 +3328,7 @@ def get_encoding(stream):
             write_debug('Plugins: %s' % [
                 '%s%s' % (klass.__name__, '' if klass.__name__ == name else f' as {name}')
                 for name, klass in itertools.chain(plugin_extractors.items(), plugin_postprocessors.items())])
+            write_debug('Plugin directories: %s\n' % plugin_directories())
```

Comment:
not sure this is even needed. But we can decide that at the end
```suggestion
            write_debug('Plugin directories: %s' % plugin_directories())
```